api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/PersonTranslatorImpl.java

```diff
@@ -69,6 +72,23 @@
 		person.setActive(!openmrsPerson.getVoided());
 		person.setBirthDate(openmrsPerson.getBirthdate());
 		
+		if (openmrsPerson.getBirthdateEstimated() != null) {
```

Comment:
Doesn't this potentially set birthdate twice?

Reply:
Initially it will set the birthday as usual but if the birthday is estimated then it will set the new birthday as expected.

Reply:
I have put all this in if else statement. it will check whether the birthday is estimated or not and according to this we will set the birthday.